modules/shared/src/main/scala/org/tesselation/schema/cluster.scala

```diff
@@ -19,6 +19,20 @@ object cluster {
   @derive(decoder, encoder, show)
   case class PeerToJoin(id: PeerId, ip: Host, p2pPort: Port)
 
+  @derive(decoder, encoder, show)
+  case class InternalTrustUpdate(id: PeerId, trust: Double)
+
+  @derive(decoder, encoder, show)
+  case class InternalTrustUpdateBatch(updates: Seq[InternalTrustUpdate])
```

Comment:
please use `List` or `Set`, instead of `Seq`. We're using `List` all over the codebase, firstly to avoid ton of conversions, secondly, it nicely interops with cats and allows for more operations.
I think `List` is more proper here, if there can be at least two updates with the same id and trust